app/src/main/java/com/dessalines/rankmyfavs/ui/components/favlist/TierListScreen.kt

```diff
@@ -191,17 +275,68 @@ fun TierList(tierList: Map<String, List<FavListItem>>) {
         verticalArrangement = Arrangement.spacedBy(SMALL_PADDING),
     ) {
         tierList.forEach { (tier, items) ->
-            TierSection(tier, items)
+            TierSection(favListId, tier, items, editTierList, tierListViewModel)
+        }
+
+        if (editTierList) {
+            Row(
+                modifier =
+                    Modifier
+                        .fillMaxWidth()
+                        .background(Color.LightGray, RoundedCornerShape(SMALL_PADDING))
```

Comment:
Don't hardcode colors like that, check the material v3 design docs for which background color is best for what surface. Otherwise they won't work for different themes and dark modes.